pom.xml

```diff
@@ -1245,8 +1245,10 @@
 
             <dependency>
                 <groupId>org.testcontainers</groupId>
-                <artifactId>testcontainers</artifactId>
+                <artifactId>testcontainers-bom</artifactId>
                 <version>1.12.3</version>
+                <type>pom</type>
+                <scope>import</scope>
```

Comment:
cc @electrum

Reply:
FTR we only list our own modules in the bom, no other dependency will be managed by it